HardCodedStringCheckerSharp/Properties/AssemblyInfo.cs

```diff
@@ -32,5 +32,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion( "1.4.0.0" )]
-[assembly: AssemblyFileVersion( "1.4.0.0" )]
+[assembly: AssemblyVersion( "1.5.0.0" )]
+[assembly: AssemblyFileVersion( "1.5.0.0" )]
```

Comment:
It probably works. I don't know if what you said helps much without just trying it :smirk: If it doesn't get used, it wouldn't be worse than it is now